.github/workflows/nodejs.yml

```
@@ -8,7 +8,7 @@ jobs:
 
     strategy:
       matrix:
-        node: ['14', '15', '16', '17', '18',]
+        node: ['14', '15', '16', '17', '18', '19']
```

Thank you very much @TylerVigario  !

There is an error in linter with Node v15.
I think we can safely drop v15 and v17 as they already reached their end-of-life: https://github.com/nodejs/release#release-schedule 
```suggestion
        node: ['14', '16', '18', '19']
```

Also, many tools (`eslint`, `ava`...) do not support them anymore:
```
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: '@eslint/eslintrc@1.4.1',
npm WARN EBADENGINE   required: { node: '^12.22.0 || ^14.17.0 || >=16.0.0' },
npm WARN EBADENGINE   current: { node: 'v15.5.1', npm: '7.3.0' }
npm WARN EBADENGINE }
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'ava@5.2.0',
npm WARN EBADENGINE   required: { node: '>=14.19 <15 || >=16.15 <17 || >=18' },
npm WARN EBADENGINE   current: { node: 'v15.5.1', npm: '7.3.0' }
npm WARN EBADENGINE }
```